src/jitlayers.cpp

```diff
@@ -1289,6 +1289,7 @@ JuliaOJIT::JuliaOJIT()
     ES(cantFail(orc::SelfExecutorProcessControl::Create())),
     GlobalJD(ES.createBareJITDylib("JuliaGlobals")),
     JD(ES.createBareJITDylib("JuliaOJIT")),
+    ExternalJD(ES.createBareJITDylib("JuliaExternal")),
```

Comment:
Long-term we might want many ExternalJDs and was considering a design where we had one JD per world in Julia.

Reply:
We can add as many JDs as we want with the ExecutionSession. The difference is that this one is linked in by default.